tests/components/media_player/test_samsungtv.py

```diff
@@ -301,3 +306,57 @@ def test_turn_on(self):
         self.device._mac = "fake"
         self.device.turn_on()
         self.device._wol.send_magic_packet.assert_called_once_with("fake")
+
+
+@pytest.fixture
+def samsung_mock():
+    """Mock samsungctl."""
+    with patch.dict('sys.modules', {
+        'samsungctl': MagicMock(),
+    }):
+        yield
+
+
+async def test_play_media(hass, samsung_mock):
+    """Test for play_media."""
+    asyncio_sleep = asyncio.sleep
+    sleeps = []
+
+    async def sleep(duration, loop):
+        sleeps.append(duration)
+        await asyncio_sleep(0, loop=loop)
+
+    def async_add_job(target, *args):
```

Comment:
I don't think we need this now that we mock `device.send_key`. We can let it use the real `async_add_job`.